.editorconfig

```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
+root = true
+
+[*.json]
+indent_style = space
+indent_size = 4
```

Comment:
We could now have 2 spaces for json files.

Reply:
@shamasis Yes, all such rectifications will occur in subsequent pull requests.